src/bin/jp2/opj_compress.c

```diff
@@ -1535,6 +1538,31 @@ static void info_callback(const char *msg, void *client_data) {
     fprintf(stdout, "[INFO] %s", msg);
 }
 
+OPJ_FLOAT64 opj_clock(void) {
```

Comment:
I would be happy to re-use the code, but opj_clock is not exported by the library, so sadly I don't have access to it from opj_(de)compress..